domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Revurdering.kt

```diff
@@ -0,0 +1,79 @@
+package no.nav.su.se.bakover.domain.behandling
+
+import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.domain.NavIdentBruker.Saksbehandler
+import no.nav.su.se.bakover.domain.beregning.Beregning
+import no.nav.su.se.bakover.domain.beregning.Beregningsgrunnlag
+import no.nav.su.se.bakover.domain.oppdrag.simulering.Simulering
+import no.nav.su.se.bakover.domain.oppgave.OppgaveId
+import java.util.UUID
+
+enum class RevurderingsStatus {
+    OPPRETTET,
+    BEREGNET,
+    SIMULERT,
+    TIL_ATTESTERING
+}
+
+sealed class Revurdering : Saksbehandling() {
+    abstract val id: UUID
+    abstract val status: RevurderingsStatus
```

Comment:
Vi bør ikke trenge en status i domenelaget når vi har en sealed class. Den kan mappes i integrasjonslagene (web,db,client). Eller hvis vi er litt late, kan vi legge den som en String istedet for enumen.

Reply:
true true, legger det in i weblagret istedet 😊